tests/test_load_image.py

```diff
@@ -12,29 +12,36 @@
 load_image_param = [
     # case 1: just filename of file in current directory.
     # expect function loads tiff file from cwd
-    "KFe2As2-00838.tif",
+    ("KFe2As2-00838.tif", Mock(fliphorizontal=False, flipvertical=False), 0),
```

Comment:
I wote a whole long description of how to do this more correctly by my browser seems to have lost it.  Annoying.  Basically we want a structure like:
```
(["KFe2As2-00838.tif", False, False)], [17, 35, 56, 78])
```
and in the function we want
```
loader = LoadImage(fliphorizontal=inputs[1], flipvertical=inputs[2])
loader.load_image(inputs[0])
```
and for the asserts
```
expected_points_1 = expecteds[0]
```
and so on.  We will need as many points as needed to correctly check the different image flips.